### ISSUE TITLE
atsec/5 - Section 7 - ITSEF description of actions to perform

### ISSUE THREAD

**Issue by woodbe** [DSC-iTC/cPP#5]
### Comment:
It`s unclear why the ITSEF is expected to perform ASE and ALC SARs from the CEM only while table 13 lists other SARs.

### Proposal:
Please clarify the expected SARs with consistency.

### Priority: 1

**Comment by woodbe** [comment]
It isn't clear to me that this is urgent, though it could be.

**Comment by woodbe** [comment]
Preferred way to go is to fix this is v1.1, but if the scheme has a concern, we can issue a TD. @robertmclemons @jfisherbah @jfmyerrors please confirm either way.

**Comment by jfisherbah** [comment]
I think the point of that statement was to say that the ASE and ALC evidence is evaluated as specified by the CEM because the cPP doesn't augment or interpret the CEM workunits for those classes, unlike the other classes which have additional guidance specifically in the context of this PP. Table 13 lists all of the SARs defined by EAL1 because those are the SARs the ITSEF is going to have to evaluate the TOE against. It just happens to be the case that ADV, AGD, ATE, and AVA all have additional evaluation activities in section 7 that go beyond "just do what the CEM requires." It's still necessary for those CEM workunits to be done, though.

But that statement may still be misleading because the ASE evaluation does require completion of TSS evaluation activities plus evaluation of additional evidence for key management and entropy. So it would probably be more accurate to say that ALC_CMC.1 and ALC_CMS.1 are evaluated in the manner specified by the CEM, and then all of the other SARs are evaluated by doing what the CEM specifies plus the evaluation activities that are specific to the PP.

 I don't think a TD is necessary because I think the need to evaluate the SARs specified in Table 13 (augmented by the guidance in section 7.1 through 7.6) is implicit, but I can't speak for the scheme.

**Comment by woodbe** [comment]
Combine into editorial SD TD rollup

**Comment by woodbe** [comment]
I looked into what the text here may be updated to and as a comparison this is pretty much identical to what NIAP has in the PP_MD_V3.3. As an alternative I checked the PP0117 from BSI/Eurosmart and there is no similar text in that document.

I have not proposed anything here, but it isn't clear to me what exactly is needed to be changed here. At this point I would not recommend issuing a TD on this as it is based on something already in widespread use (even if it isn't completely clear). This could be revisited in v1.1 to see if something better could be written, though I think that given everything else this should be a low priority given it is common.
